win32_event_log/README.md

```diff
@@ -60,7 +60,7 @@ Depending on collection method, the channel name can be used for the following c
 
 To collect Windows Event Logs as Datadog Events, configure channels under the `instances:` section of your `win32_event_log.d/conf.yaml` configuration file.
 
-The agent can be configured to collect Windows Event Logs as Datadog Events in two ways. Each method has its own configuration syntax for channels and for filters (see [Filtering Events](?tab=events#filtering-events)). The legacy method uses WMI and is the default mode for an instance. The newer method uses the Event Log API. We recommend using the Event Log API because it has better performance. To use the Event Log API collection method, set `legacy_mode: false` in each instance.
+The agent can be configured to collect Windows Event Logs as Datadog Events in two ways. Each method has its own configuration syntax for channels and for filters (see [Filtering Events](?tab=events#filtering-events)). The legacy method uses WMI and is the default mode for an instance. The newer method uses the Event Log API. We recommend using the Event Log API because it has better performance. To use the Event Log API collection method, set `legacy_mode: false` in each instance. If `legacy_mode: false` is set, the `path` is required to be set in the `\win32_event_log.d\conf.yaml` file.
```

Comment:
cc @clarkb7

Reply:
@shanelhuang @clarkb7 Unrelated to this change, but should we mention that the legacy mode (WMI) is deprecated and will be removed in a future release, similar to the warning that is already being logged from the code: https://github.com/DataDog/integrations-core/blob/7.43.x/win32_event_log/datadog_checks/win32_event_log/legacy/win32_event_log.py#L55

Reply:
I'm not sure this is the right spot to include the new sentence about the `path` option being required. I think it is the only required option (right now) for `legacy_mode: false`. But also if `legacy_mode` is not set or set to `true` then it has it's own [required options](https://github.com/DataDog/integrations-core/blob/7c56e905fe00e686106f5922cd2c10afee810756/win32_event_log/datadog_checks/win32_event_log/legacy/win32_event_log.py#L82-L86). Maybe there could be another paragraph/section that lists the required options for each mode?

a bit broader than that, we have been thinking about how to change these docs to put more focus on the newer (newer is a misnomer now, it was added in 2020) implementation (`legacy_mode: false`), and less on the legacy mode.  Some thoughts have included
* Should there be a separate "Events (deprecated)" tab to separate the legacy mode
* Put the "# Event Log API (better performance)" and "# NON-LEGACY MODE" examples first/above the legacy ones